Managers/LuaMan.cpp

```diff
@@ -2305,7 +2305,7 @@ int LuaMan::Initialize() {
             .def("IsOfActor", &MovableMan::IsOfActor)
             .def("GetRootMOID", &MovableMan::GetRootMOID)
             .def("RemoveMO", &MovableMan::RemoveMO)
-            .def("KillAllActors", &MovableMan::KillAllActors)
```

Comment:
You're gonna have to merge development into your branch and redo this cause of maxim's luaman changes. Bug him if you've got questions.